advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/preparing_db/preparing_postgres_source_databases.mdx

```diff
@@ -0,0 +1,198 @@
+ - `<TABLE_NAME>` is the name of a table to migrate.
+
+### Grant `CREATE` privileges on the source database to the CDC migration role
+
+Since the `pgoutput` plugin is being used, the migration user needs to be granted `CREATE` priveleges on the source database in order for Debezium to be able to create publications. The 'CREATE' privelege can be granted using the following command:
+
+```sql
+GRANT CREATE ON DATABASE <PG_DB_NAME> to <MIGRATION_ROLE>
+```
+
+## SSL configuration
+
+Ensure you configure your source database server to [accept SSL connections](https://www.postgresql.org/docs/current/ssl-tcp.html) to allow the EDB DMS Reader to connect to it. You must create a server certificate and a server private key, for example, with OpenSSL, to enable this configuration.
+
+## Run the config validation script
+
+Now that you have stepped through the entire configuration process, validate your Postgres configuration using the [config validation script](#config-validation-script) with the correct parameters. To illustrate, consider the following example:
+
+1. First, create an array that contains the name of all the tables that need to be migrated in <schema_name>.<table_name> format. In this example:
+
+```bash
+arr=(test1.table1 test1.table2 test1.table3 test1.table4)
+```
+
+2. Next, run the script with the necessary parameters. In this example:
+
+```bash
+PG_USERNAME=postgres PG_PASSWORD=password PG_HOST=localhost PG_PORT=5432 DB_NAME=postgres DBZ_USERNAME=debezium DBZ_PASSWORD=dbz ./postgresConfigValidation.sh "${arr[@]}"
+```
+
+You should get a response similar to the following:
+
+```
+ *** [Transporter] - Validate WAL Level
+wal_level:  logical
+[Pass] wal_level is 'logical'.
+
+ *** [Transporter] - Validate max WAL senders
+max_wal_senders: 10
+[Pass] max_wal_senders is at least 1.
+
+ *** [Transporter] - Validate max replication slots
+max_replication_slots: 10
+[Pass] max_replication_slots is at least 1.
+
+ *** [Transporter] - Validate max WAL size
+max_wal_size: 1 GB
+[Fail] max_wal_size (1 GB) is less than 8 GB
+
+ *** [Transporter] - Validate checkpoints
+checkpoints_timed: 378
+checkpoints_req: 3
+[Pass] Timed checkpoints are more frequent than requested checkpoints.
+
+ *** [Transporter] - Check miguser user role
+[Pass] User 'miguser' is present
+[Pass] User 'miguser' has replication permission
+
+ *** [Transporter] - Check SELECT privilege on the tables to be migrated
+[Pass] User miguser has select privilege on all tables to be migrated.
+
+ *** [Transporter] - Check presence of a replication slot
+[Pass] Replication slot 'miguser' is present
+
+```
+
+!!! Note 
+In the above example output, note that the `max_wal_size` check failed. Despite this, the migration was able to be run successfully without any issues. Therefore, although this check might fail, a lower setting may actually be sufficient in certain use cases depending on the workload on the source database.
```

Comment:
Add the following text to this note to account for the replication slot failure:

> The failure message related to the replication slot can be ignored. This is an old check that is no longer needed as the cdc-reader software automatically creates and manages the needed replication slot. The check will be removed in a future version of the postgresConfigValidation.shscript.